src/Button.js

```diff
@@ -7,6 +7,8 @@ class Button extends React.Component {
         accent: PropTypes.bool,
         className: PropTypes.string,
         colored: PropTypes.bool,
+        component: PropTypes.any,
+        href: PropTypes.any,
```

Comment:
Yep but I think it's not valid for `href`. `href` has always been used as a string, so people know they should put a string in there, nothing else. If you write a custom element taking a non string `href`, maybe you should rename it ;)